help.go

```diff
@@ -46,22 +46,22 @@ module's go.mod.
 By default, gobin installs the main packages to $GOBIN (or $GOPATH/bin if GOBIN
 is not set, which defaults to $HOME/go/bin if GOPATH is not set).
 
-The -r flag takes exactly one main package argument and runs that package.  It
-is similar therefore to go run. Any arguments after the single main package
+The -run flag takes exactly one main package argument and runs that package.
+It is similar therefore to go run. Any arguments after the single main package
 will be passed to the main package as command line arguments.
 
 The -p flag prints the cache install path for each of the provided packages
 once versions have been resolved.
 
-The -r and -p flags are mutually exclusive.
+The -run and -p flags are mutually exclusive.
 
-The -g flag forces gobin to perform a network fetch for the provided main
+The -u flag forces gobin to perform a network fetch for the provided main
```

Comment:
This isn't quite accurate, I think, as it won't necessarily perform a network fetch. Perhaps something like:

    If the `-u` flag is specified, gobin will ensure that the most current module satisfying
    the specified version is built, by checking the network if necessary. For example,
    `gobin -u $module@latest` will always ensure that `$module` is updated to the latest
    published version, whereas without the `-u` flag, any currently built binary satisfying that
    version will be used.

(probably still not accurate, but better?)

Reply:
Yes, good point. Do you think it's worth adding an "example" here that `gobin -u $module@v1.0.0` will not hit the network if v1.0.0 is available locally? To drive home the point about "if necessary"?